src/electron/office/document_client.cc

```diff
@@ -771,4 +773,11 @@ bool DocumentClient::SendContentControlEvent(
 
   return true;
 }
+
+v8::Local<v8::Value> DocumentClient::As(const std::string& type, v8::Isolate* isolate)
+{
+  void *component = document_->getXComponent();
+  return convert::As(isolate, component, type);
```

Comment:
If you put in garbage here does electron do the error handling or does the program crash?

Reply:
There is no error, it returns undefined.